build.bash

```diff
@@ -240,16 +246,18 @@ if [ "$hw_platform" == "pi-raspios32" ] || [ "$hw_platform" == "pi-raspios64beta
   fi
   check_command_availability_and_exit "$REQ_COMMANDS" "$REQ_PACKAGES"
 
-  if [ -f "$zipfile" ]; then
-    echo_process "Using local copy of Raspberry Pi OS ($bits-bit) Lite image, $zipfile... "
-    cp $zipfile "$buildfolder/$zipfile"
+
+  if [[ $zipfile == "$(ls -- "$findfile" 2> /dev/null)" ]]; then
```

Comment:
@ecdye  I don't understand why github does not show our old conversation. I write on based on the notifications I got via email....

I do not see how this helps. Why do we need the ls at all? `$zipfile` is specific. `ls ...` is not (since I typically find a bunch of files with this pattern in my build dir.

Before the change, the semantics of caching process were: download the current image and store it under the name `raspios_lite_armhf/arm64_latest.zip`. Either we go back to this behavior, or we decide to use the filename from `$zipfile`. That is what you implemented. But then let's use `if [ -f $zipfile ]`....

Reply:
Lets go with your solution, I think I understand what you mean.

Reply:
@holgerfriedrich like that?